README.md

```diff
@@ -61,23 +61,23 @@ which make development much easier.
 
 These include:
 
-* PostgreSQL 12 (10 probably works)
-* NodeJS 14 (we require 14 because we want the full internationalization built-in)
-* Ruby 2.7.4
+- PostgreSQL 12 (10 probably works)
+- NodeJS 14 (we require 14 because we want the full internationalization built-in)
+- Ruby 2.7.4
```

Comment:
All of these formatting changes aren't really related to the change so please remove them.